src/db/repository/postgres/AgenciesRepository.ts

```diff
@@ -0,0 +1,64 @@
+import { Kysely, UpdateObject } from 'kysely';
+
+import { Agencies, DB } from '../../types/generated/database';
+
+export type AgenciesUpdateParams = Omit<
+	UpdateObject<DB, 'agencies'>,
+	'id' | 'created_at' | 'updated_at'
+>;
+export type AgenciesCreateParams = Omit<
+	Agencies,
+	'id' | 'is_verified' | 'created_at' | 'updated_at' | 'deleted_at'
+>;
+
+export default class AgenciesRepository {
+	constructor(private readonly database: Kysely<DB>) {}
+
+	getByUserId(id: string) {
```

Comment:
```suggestion
	getByAccountManagerId(id: string) {
```

I don't think we should rely on the input variable being descriptive enough to clarify that it is the ID of the account manager.